cartpole_v0: #namespace

        #qlearn parameters
        alpha: 0.03
        alpha_decay: 0.01
        gamma: 1.0
        epsilon: 1.0
        epsilon_decay: 0.995
        epsilon_min: 0.01
        batch_size: 64 #max size of the batches sampled from the memory
        episodes: 500
        n_win_ticks: 100
        min_episodes: 100
        monitor: True
        quiet: False


        control_type: "velocity"
        min_pole_angle: -0.7
        max_pole_angle: 0.7
        max_base_velocity: 50
        max_base_pose_x: 1.0
        min_base_pose_x: -1.0

        n_states: 4
        n_actions: 2

        #  openai-gym specific parameter
        running_step : 0.03 #amount of time the control will be executed.
        pos_step: 1.0 #increment in position/velocity/effort, depends on the control for each command
        init_pos: 0.0
        wait_time: 0.1
